pkg/collector/corechecks/snmp/gosnmplib/gosnmp_value.go

```diff
@@ -32,7 +32,7 @@ func GetValueFromPDU(pduVariable gosnmp.SnmpPDU) (string, valuestore.ResultValue
 	case gosnmp.OctetString, gosnmp.BitString:
 		bytesValue, ok := pduVariable.Value.([]byte)
 		if !ok {
-			return name, valuestore.ResultValue{}, fmt.Errorf("oid %s: OctetString/BitString should be []byte type but got %T type: %#v", pduVariable.Name, pduVariable.Value, pduVariable)
+			return name, valuestore.ResultValue{}, fmt.Errorf("oid %s: OctetString/BitString should be []byte type but got type `%T` and value `%v`", pduVariable.Name, pduVariable.Value, pduVariable.Value)
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
I prefer `reflect.TypeOf(pduVariable.Value)`, did not know `%T`